lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -196,17 +196,25 @@ public JourneyResponse handleRequest(JourneyRequest event, Context context) {
                 }
             }
 
-            if (userHasFaceToFaceRequest && Objects.isNull(faceToFaceVc)) {
-                var message =
-                        new StringMapMessage()
-                                .with(
-                                        LOG_MESSAGE_DESCRIPTION.getFieldName(),
-                                        "F2F cri pending verification.");
-                LOGGER.info(message);
-                return new JourneyErrorResponse(
-                        JOURNEY_ERROR_PATH,
-                        HttpStatus.SC_INTERNAL_SERVER_ERROR,
-                        ErrorResponse.PENDING_VERIFICATION_EXCEPTION);
+            if (userHasFaceToFaceRequest) {
+                if (Objects.isNull(faceToFaceVc)) {
+                    var message =
+                            new StringMapMessage()
+                                    .with(
+                                            LOG_MESSAGE_DESCRIPTION.getFieldName(),
+                                            "F2F cri pending verification. No VC to delete.");
+                    LOGGER.info(message);
+                } else {
+                    userIdentityService.deleteVcStoreItems(userId);
+                    var message =
+                            new StringMapMessage()
+                                    .with(
+                                            LOG_MESSAGE_DESCRIPTION.getFieldName(),
+                                            "F2F cri pending verification. Existing F2F VC deleted.");
+                    LOGGER.info(message);
+                }
+
+                return JOURNEY_NEXT;
```

Comment:
To preserve the existing log and audit message below I think all we need to do here is log that there's a pending F2F request then let it continue